scenarios/arcaflow/kill-pod/input.yaml

```diff
@@ -0,0 +1,7 @@
+kubeconfig: {}
+kill: 3
```

Comment:
We might want to set the default to 1 to be less aggressive, also we might not always have 3 pods by default.